main.cpp

```diff
@@ -14,6 +14,8 @@
 int main(int argc, char *argv[])
 {
     QCoreApplication app(argc, argv);
+    if (argc > 2 && !strcmp(argv[1], "--removeUserFiles"))
```

Comment:
I'd much prefer seeing some sort of more or less conforming arg parser in here ;-)

Reply:
Sure needed if there ever comes more arguments to parse. Not compilicating things for now as there in only this one simple case.